{"project": "spacy", "bug": 10, "project_url": "https://github.com/explosion/spaCy", "file_changed": "spacy/errors.py", "buggy_commit_id": "38de08c7a99d5d8c490223126071afe7dd4f4b67", "fixed_commit_id": "52904b72700a3f301a26563d3f94493bad96a446", "lines_deleted": {}, "lines_added": {"478": "    E171 = (\"Matcher.add received invalid on_match callback argument: expected \"\n", "479": "            \"callable or None, but got: {arg_type}\")\n"}}